packages/lib/src/vis/tiles/hooks.ts

```diff
@@ -1,13 +1,34 @@
 import { useThree } from '@react-three/fiber';
+import { clamp } from 'lodash';
 
 import { useVisibleDomains } from '../hooks';
+import { useAxisSystemContext } from '../shared/AxisSystemContext';
+import { normalize } from '../utils';
 
-export function useDataPerPixel() {
+export function useNormalizedVisibleDomains() {
   const { xVisibleDomain, yVisibleDomain } = useVisibleDomains();
+
+  const { abscissaConfig, ordinateConfig } = useAxisSystemContext();
+  const { visDomain: abscissaVisDomain } = abscissaConfig;
+  const { visDomain: ordinateVisDomain } = ordinateConfig;
+
+  return {
+    xNormVisibleDomain: xVisibleDomain.map((v) =>
+      clamp(normalize(...abscissaVisDomain)(v), 0, 1)
```

Comment:
Suggestion: you could use a `ScaleLinear` here instead of normalizing "by hand"

Reply:
Yes, why not, I used visx's scaleLinear